examples/cce/find_cluster_node.py

```diff
@@ -19,8 +19,8 @@
 conn = openstack.connect(cloud='otc')
 
 
-cluster = "cluster_name_or_id"
+cluster = 'name_or_id'
 cluster = conn.cce.find_cluster(cluster)
-node = "node_name_or_id"
+node = 'name_or_id'
```

Comment:
you can insert name right into find_cluster_node

Reply:
Yes I can, but I used this simpler form to distinguish in those example file, where parameters are coming from. Nevertheless, I changed it as requested.